File: Atividade 1/Exercício 3.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    float salario_atual, novo_salario;
    int ajuste;
    printf("Digite o salario atual: \n");
    scanf("%f", &salario_atual);
    printf("Para aumento, digite 1, reducao, 2: \n");
    scanf("%d", &ajuste);
    if (ajuste == 2){
        novo_salario = salario_atual * 0.9;
    }
    else novo_salario = salario_atual * 1.1;
    printf("Novo salario eh %.2f\n", novo_salario);    
    return 0;
}

File: Atividade 1/Exercício 3.py
salario_atual = float(input("Digite o salário atual: "));
while(True):
    ajuste = int(input("Para aumento 1, para redução 2: "));
    if ajuste == 1 or ajuste == 2:
        break;
    else:
        print("Opção inválida, tente novamente.");
if ajuste == 1:
    novo_salario = salario_atual * 1.1;
else:
    novo_salario = salario_atual * 0.9;
print(f'O novo salário é {novo_salario:.2f}');